packages/components/src/internal/components/domainproperties/models.tsx

```diff
@@ -77,6 +91,12 @@ export interface FieldDetails {
     ontologyLookupIndices: number[];
 }
 
+export interface SummaryGrid {
```

Comment:
This `SummaryGrid` seems misleading. Isn't this more related to the grid columns? Also, I'd probably be a bit more descriptive in the title and include "DomainProperties" in the name. Maybe something like "DomainPropertiesGridColumn"?

Reply:
Ah, good point. Done!